src/cljc/athens/parser/impl.cljc

```diff
@@ -98,6 +99,7 @@ strikethrough = <#'~~(?!\\s)'>
 
 link = md-link
 image = <'!'> md-link
+audio = <'^'> md-link
```

Comment:
I just did some copy and paste, but I don't fully understand how the interpreter works